libtransmission/quark.cc

```diff
@@ -78,6 +78,7 @@ auto constexpr MyStatic = std::array<std::string_view, TR_N_KEYS>{
     "creator"sv,
     "cumulative-stats"sv,
     "current-stats"sv,
+    "daemon-startPaused"sv,
```

Comment:
_(nit)_

Maybe `start_paused` here?

- I don't have any plans to add this to the other clients, but no need to limit the key specifically to `daemon`
- RPC variable naming is [currently a mess](https://github.com/transmission/transmission/issues/2855) but in general we should probably use `snake_case` names going forward since it's slightly friendlier to client code, e.g. JavaScript objects can have literals with underscores but not hyphens (`o.start_paused` instead of `o["start-paused"]`)

Reply:
719faee63105bc2a3572855fb882298daa4ef202

Reply:
>"Predefined quarks must be sorted by their string value"

This rule is causing issues. :/

Reply:
462ec6d08ff09605b752a04490e0eb2a6d103511

Reply:
> RPC variable naming is https://github.com/transmission/transmission/issues/2855

/offtopic

@ckerr Maybe we can at least normalize it then?
Make everything `snake_case` internally and normalize any external input with simple `std::replace(call_name.begin(), call_name.end(), '-', '_')`.

Reply:
@nevack let's discuss in 2855 instead of here :smile_cat: